src/ui/public/vislib/visualizations/pie_chart.js

```diff
@@ -32,7 +33,7 @@ export function VislibVisualizationsPieChartProvider(Private) {
 
       const charts = this.handler.data.getVisData();
       this._validatePieData(charts);
-
+      this.ABBREVIATIONS = ['', 'k', 'Mil', 'Bn', 'Tn'];
```

Comment:
I think we should rather use the common [unit prefixes](https://en.wikipedia.org/wiki/Unit_prefix) as in other places, i.e. `k`, `M`, `G`, `T`, `P`. 

These are imho better understand and less prone to misunderstanding due to cultural/language differences. 

*Example time:* In German we have a bit different system to naming millions and above:

* Million [EN] -> Million [DE]
* Billion [EN] -> Milliarde [DE]
* Trillion [EN] -> Billion [DE]
* Quadrillion [EN] -> Billiarde [DE]
* Quintillion [EN] -> Trillion [DE]
* ...

Even while speaking English most of the time, I have a hard time, thinking when seeing a billion or Bn about an English billion rather than a German billion. Also `Tn` could be easily misunderstood as the abbreviation of "Tausend" (the German word for thousand).

In general using the well acknowledged prefixes, would imho be a better solution and better aligned with the rest of Kibana.